package.json

```diff
@@ -25,7 +25,7 @@
     "lerna": "^3.13.1",
     "lint-staged": "^8.1.5",
     "prettier": "^1.16.4",
-    "semantic-release": "~15.9.0",
+    "semantic-release": "~15.13.3",
```

Comment:
Version 15.9.0 is required. Need to figure out if I can either remove this change, or accept it, revert it, and then let greenkeeper open another PR to fix it that I leave.